server/cshr/celery/task.py

```diff
@@ -0,0 +1,86 @@
+from django.conf import settings
+
+from celery import Celery
+from celery.schedules import crontab
+import os
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "server.settings")
+app = Celery("tasks", broker="redis://localhost:6000")
```

Comment:
we have to set this as env var.
```suggestion
app = Celery("tasks", broker=config("REDIS_HOST))
```